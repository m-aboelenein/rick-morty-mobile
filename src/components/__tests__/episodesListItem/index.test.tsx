import * as React from 'react';
import EpisodesListItem from '../../episodesListItem';
import renderer from 'react-test-renderer';

it('renders an episode', () => {
  const tree = renderer.create(
    <EpisodesListItem
      episode={{
        id: '1',
        name: 'pickle Rick',
        air_date: 'january 1st 2018',
      }}
    />,
  );
  expect(tree).toMatchSnapshot();
});
