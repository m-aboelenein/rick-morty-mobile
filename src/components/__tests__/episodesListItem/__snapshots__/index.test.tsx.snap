// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders an episode 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "borderRadius": 8,
      "flexDirection": "row",
      "height": 40,
      "justifyContent": "space-between",
      "margin": 8,
      "paddingHorizontal": 16,
    }
  }
>
  <Text
    numberOfLines={1}
    style={
      Object {
        "color": "black",
        "flex": 1,
        "fontSize": 13,
      }
    }
  >
    pickle Rick
  </Text>
  <Text
    numberOfLines={1}
    style={
      Object {
        "color": "green",
        "fontSize": 14,
      }
    }
  >
    january 1st 2018
  </Text>
</View>
`;
